### ISSUE TITLE
Noisy Output: Could not create directory "plantuml-images" 

### ISSUE THREAD

**Issue by lixueyi83** [timofurrer/pandoc-plantuml-filter#29]
I am continue getting this noisy message, Could not create directory "plantuml-images", when the plantuml-images folder was already created previously. 

This is the command I convert markdown to pdf 

```bash
    pandoc --filter=pandoc-plantuml test.md -o test.pdf
```

![image](https://github.com/timofurrer/pandoc-plantuml-filter/assets/25063985/8bdd6e55-4958-4a5f-8ed7-1162fcd4caad)

There are a couple of plantuml in the test.md file, run the above command twice. It seems for the first plantuml diagram, pandoc will create the folder since it's not there... But when pandoc tries to convert the 2nd plantuml diagram, it still trying th create the folder and then it starts complaining.

**Comment by fliiiix** [comment]
yeah that is this issue https://github.com/timofurrer/pandoc-plantuml-filter/issues/25 

i know i just didn't had the time and motivation to fix it 😅

**Comment by Felixoid** [comment]
But it's the https://github.com/jgm/pandocfilters/blob/5c6c1954401099f64e36e0fa52e29b27a52f581b/pandocfilters.py#L52 issue.

And it should be addressed in the pandocfilters https://github.com/jgm/pandocfilters/pull/109

**Comment by Felixoid** [comment]
Well, actually, in the case of @lixueyi83 it's an outdated pandocfilters https://github.com/jgm/pandocfilters/commit/4da2c73331c94d1227ea5104b0bdcad4c1e1ffda

But still, the current code is supposed to spam the error somehow `¯\_(ツ)_/¯`
